etc/config.sample.toml

```
@@ -111,6 +111,10 @@
   # to cache snapshotting.
   # max-concurrent-compactions = 0
 
+  # MaxConcurrentDeletes is the maximum number of simultaneous DELETE calls on a shard
+  # The default is 1, which was the previous hard-coded value.
```

This is user-facing documentation. Better might be:
```suggestion
  # The default is 1, and should be left unchanged for most users
```